tests/estimation/test_estimate_msm.py

```diff
@@ -176,3 +176,24 @@ def test_to_pickle(tmp_path):
     )
 
     calculated.to_pickle(tmp_path / "bla.pkl")
+
+
+def test_caching():
```

Comment:
To test caching we should use an identity check (via `is`), not almost equal with very high precision.